java/src/jmri/jmrit/display/layoutEditor/LayoutTurnout.java

```diff
@@ -2003,7 +2001,7 @@ protected int findHitPointType(@Nonnull Point2D hitPoint, boolean useRectangles,
         Point2D p, minPoint = MathUtil.zeroPoint2D;
 
         double circleRadius = LayoutEditor.SIZE * layoutEditor.getTurnoutCircleSize();
-        double distance, minDistance = POSITIVE_INFINITY;
+        double distance, minDistance = Float.POSITIVE_INFINITY;
```

Comment:
There is a type mismatch here.  The value is a double, but you are assigning a Float value.  You should replace Float.POSITIVE_INFINITY with Double. POSITIVE_INFINITY

Reply:
Thanks for the catch… fixed.